web/app/js/components/Namespace.jsx

```diff
@@ -126,7 +126,7 @@ class Namespaces extends React.Component {
     return (
       <div className="page-content">
         { !this.state.error ? null : <ErrorBanner message={this.state.error} /> }
-        { !this.state.loaded ? <ConduitSpinner /> : (
+        { !this.state.loaded ? <Spin size="large" /> : (
```

Comment:
if there's something larger than `large`, i'd say go for it.

Reply:
haha, no large is the biggest it gets!
https://ant.design/components/button/
This is already bigger than the existing conduit spinner though!